pkg/cmd/cli/common.go

```diff
@@ -18,11 +18,65 @@ package cli
 
 import (
```

Comment:
I'd probably rename this file to `delete_options.go` since that's now what it contains.